app/views/items/_form.html.erb

```diff
@@ -1,10 +1,12 @@
-<%= simple_form_for(@item) do |f| %>
+<%= simple_form_for (@item), :html => {:multipart => true} do |f| %>
```

Comment:
Ruby can have trouble if there's a space between a method call and the parentheses around its argument(s).  Unless you are writing it the exact way you did for a reason, I'd recommend using either parentheses _or_ space but not both.  Either of these should work:

```ruby
<%= simple_form_for @item, :html => {:multipart => true} do |f| %>
<%= simple_form_for(@item, :html => {:multipart => true}) do |f| %>
```
(I'm guessing that "@item" and ":html => ..." are both parameters to the `simple_form_for` method.  If so, they should both be inside the parentheses if you use them.)